packages/jetpack-redirects/README.md

```diff
@@ -0,0 +1,54 @@
+# Jetpack Redirects
+
+A helper function to build URLs using the `jetpack.com/redirect` service.
+
+## Usage
+
+`getRedirectUrl( required source, optional args );`
+
+### $source (required)
+
+Source can be either a “source handler” or a URL.
+
+A “source handler” must be registered in the Jetpack Redirects service, on the server side. It’s a slug that points to an URL that may or may not have dynamic parts in it.
+
+A “URL” is a string that must start with “https://" and doesn’t need to be registered on the server. However, if it is registered, it will point to the URL set as target there rather than to the source. (Note: It will only work for whitelisted domains)
```

Comment:
I don't understand the part about `https://*` strings being registered. Such a string can be registered on the server as source?

Reply:
Yes, that's one of the mais reasons why to use redirects.

This enables us to fix broken links even for people using an outdated version of the Jetpack plugin. If all URLs are added using this function, we can point links to different locations on the server side, without the need of a new version to fix it.

Reply:
> Note: It will only work for whitelisted domains

This applies to the `https://*` sources that are registered, not registered, or all?

I understand the case for unregistered URLs -- if the redirect endpoint agreed to redirect to any URL it gets as param, it would probably be a security vulnerability.